charts/elasticsearch-umbrella/charts/elasticsearch-statefulset/templates/statefulset.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     {{- include "statefulset.labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
+  {{- with .Values.podManagementPolicy }}
+  podManagementPolicy: {{ . }}
```

Comment:
Another tricky thing from Elastic 7.X. For Elastic 6.x it's Ok to start master nodes one by one, but in Elastic 7.X it's required to start them all at the same time to facilitate the autodiscovery. It's only required by master nodes. For master nodes we will need to set "Parallel" in the "values". Since it's only required for Elastic 7.X we will keep it empty for now and we can safely delegate in the child charts to set this value.